applications/asset_tracker_v2/doc/location_module.rst

```diff
@@ -76,7 +76,7 @@ The following is an example of the CLI command:
 
 .. code-block:: console
 
-   west build -p -b nrf9160dk_nrf9160ns -- -DSHIELD=nrf7002ek_nrf7002 -DDTC_OVERLAY_FILE=nrf9160dk_with_nrf7002ek.overlay -DOVERLAY_CONFIG=overlay-nrf7002ek-wifi-scan-only.conf
```

Comment:
Line 74 also needs update. There are also many other references to nrf7002ek_nrf7002 across NCS. Just do a search and replace across the tree :)